scipy/stats/tests/test_stats.py

```diff
@@ -3463,6 +3463,8 @@ def test_argument_checking(self):
         assert_raises(ValueError, stats.ks_2samp, [1], [])
         assert_raises(ValueError, stats.ks_2samp, [], [])
 
+
+    @pytest.mark.slow
```

Comment:
@pvanmulbregt this test was taking 1.64s on my machine. Is it OK to mark it slow?

Reply:
Should be fine